client/components/select-dropdown/index.jsx

```diff
@@ -218,6 +228,7 @@ class SelectDropdown extends Component {
 				>
 					<div id={ 'select-dropdown-' + this.instanceId } className="select-dropdown__header">
 						<span className="select-dropdown__header-text" aria-label={ this.props.ariaLabel }>
+							{ selectedExtra && <>{ selectedExtra }</> }
```

Comment:
This seems redundant. `selectedExtra` shouldn't render anything when _falsy_, so I think this can be simplified?
```suggestion
							{ selectedExtra }
```

Reply:
Fixed